cmds/cpio/fs_unix.go

```diff
@@ -0,0 +1,49 @@
+// Copyright 2013-2017 the u-root Authors. All rights reserved
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package main
+
+import (
+	"fmt"
+	"io"
+	"os"
+)
+
+func create(f *File) error {
+
+	m, err := cpioModetoMode(f.Mode)
+	if err != nil {
+		return err
+	}
+
+	switch m {
+	case os.ModeSocket:
```

Comment:
Go allows commas in switches, so you can go:

```
case os.ModeSocket, os.ModeDevice, os.ModeCharDevice:
    return fmt.Errorf(...)
```

And combine a few of these cases into 1.